### PR TITLE
Item 6174: R saveRuns with R doc

### PR DESCRIPTION
**Pull request by labkey-ankurj**

### CODE REVIEW

File: Rlabkey/R/labkey.experiment.R [hunk 1]
```diff
@@ -171,3 +171,30 @@ labkey.experiment.saveBatch <- function(baseUrl=NULL, folderPath, assayConfig =
...
+
+    ## Validate required parameters
+    if (missing(folderPath)) stop (paste("A value must be specified for folderPath."))
+    if (missing(protocolName)) stop (paste("A value must be specified for protocolName."))
+    if (missing(runList)) stop (paste("A value must be specified for runList."))
+
+    ## normalize the folder path
+    folderPath <- encodeFolderPath(folderPath)
+
+    ## Now post form with batch object filled out
```
Comment (cnathe) [inline Rlabkey/R/labkey.experiment.R]: Very minor but this comment should probably be updated since it doesn't deal with "batch".

**Review by cnathe — APPROVED** [review]

File: Rlabkey/R/labkey.experiment.R (hunk 1 above)
Comment (labkey-ankurj) [reply Rlabkey/R/labkey.experiment.R]: Done

File: Rlabkey/man/labkey.experiment.saveRuns.Rd [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+\donttest{
+
+library(Rlabkey)
+
+## example with datarows
+
+df <- data.frame(participantId=c(1:3), visitId = c(10,20,30), sex = c("f", "m", "f"))
+run <- labkey.experiment.createRun(list(name="new assay run"), dataRows = df)
+labkey.experiment.saveRuns(baseUrl="http://labkey/", folderPath="home",
+    assayConfig=list(assayName="GPAT", providerName="General"), runList=run)
```
Comment (cnathe) [inline Rlabkey/man/labkey.experiment.saveRuns.Rd]: See my last email. It sounds like maybe this example needs to also be removed? Either that or we need updated info in the parameters section of this doc as to what params to use to upload an assay run.

**Review by cnathe — CHANGES_REQUESTED** [review]

**Review by cnathe — APPROVED** [review]

File: Rlabkey/R/labkey.experiment.R [hunk 3]
```diff
@@ -171,3 +171,30 @@ labkey.experiment.saveBatch <- function(baseUrl=NULL, folderPath, assayConfig =
...
+    ## Validate required parameters
+    if (missing(folderPath)) stop (paste("A value must be specified for folderPath."))
+    if (missing(protocolName)) stop (paste("A value must be specified for protocolName."))
+    if (missing(runList)) stop (paste("A value must be specified for runList."))
+
+    ## normalize the folder path
+    folderPath <- encodeFolderPath(folderPath)
+
+    ## Now post form with runs object filled out
+    url <- paste(baseUrl, "assay", folderPath, "saveAssayRun.api", sep="")
```
Comment (cnathe) [inline Rlabkey/R/labkey.experiment.R]: I get the following error when trying to test out this new Rlabkey function locally. Is there a typo here? Should this be "saveAssayRuns.api"?

Screen Shot 2019-09-16 at 10 11 05 AM

**Review by cnathe — CHANGES_REQUESTED** [review]

File: Rlabkey/R/labkey.experiment.R (hunk 3 above)
Comment (labkey-ankurj) [reply Rlabkey/R/labkey.experiment.R]: oh dang! I forgot to change it here.. I changed the action name from SaveRun to SaveRuns last Friday. 
Thanks for catching it.

**Review by cnathe — APPROVED** [review]

**Pull request merged**